### ISSUE TITLE
Bug: Opens too many files

### ISSUE THREAD

**Issue by ad-si** [kamalsacranie/anki-panky#4]
I'm getting this error for a deck with ~ 300 images:

```txt
Rendering deck: "Flashcards"
anki-panky: /Users/adrian/Projects/Flashcards/./images/test.jpg: openBinaryFile: resource exhausted (Too many open files)
```

One workaround is to run:
```sh
ulimit -n 512
anki-panky cards.md
```
but ideally the code shouldn't open all the files simultaneously.

**Comment by kamalsacranie** [comment]
Thanks for pointing this out. Could you maybe put the deck on a repo so I could clone it and test on it while im fixing the issue? Thanks

**Comment by kamalsacranie** [comment]
Think this is fixed by #7. I've re compiled the release 0.0.0.2 binary. Will you give it a shot and let me know if things work out.

**Comment by ad-si** [comment]
Awesome, seems to work fine now! Thanks!
